operations/admin_all_configs.go

```diff
@@ -79,12 +78,7 @@ func fetchAndWriteConfig(c *legacyClient, project string) error {
 		return errors.Wrapf(err, "failed to fetch config for project %s, version %s", project, versions[0])
 	}
 
-	data, err := yaml.Marshal(config)
-	if err != nil {
-		return errors.Wrapf(err, "failed to marshal configuration for project %s", project)
-	}
-
-	err = ioutil.WriteFile(project+".yml", data, 0666)
+	err = ioutil.WriteFile(project+".yml", config, 0666)
```

Comment:
0644, typically

Reply:
oh, you didn't make this change, but it's tiny enough that it's worth fixing here

Reply:
Done